test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/utils/ChatContainer.spec.js

```diff
@@ -184,6 +184,31 @@ describe("ChatContainer", () => {
         expect(secondCallToSendDataPass).toMatchObject({ testDataPass: 'worked' });
     });
 
+    it("Mix: process external links, with error returned from JsonUtils bacause of malformed datapass test", () => {
+        chatContainer = new ChatContainer(null, null, mockSDK);
+
+        const externalLinkEventWithInvalidJson = {
+            target : {
+                dataset: {
+                    "nuanceMessageText": "Where do you live?",
+                    "nuanceDatapass": "{'testDataPass': worked'}",//missing ' before worked
```

Comment:
took me a minute to wrap my head around this one, perhaps defining a const above this object would be a bit clearer? `const invalidJson = "{'testDataPass': worked'}"`